Add support for Mongoose@6

```
npm i mongoose-update-versioning
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: test-pacakge@1.0.0
npm ERR! Found: mongoose@6.0.12
npm ERR! node_modules/mongoose
npm ERR!   mongoose@"^6.0.12" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer mongoose@"5.x" from mongoose-update-versioning@0.1.1
npm ERR! node_modules/mongoose-update-versioning
npm ERR!   mongoose-update-versioning@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /Users/xxx/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/xxx/.npm/_logs/2021-10-27T09_00_17_910Z-debug.log

```

--- a/package.json
+++ b/package.json
@@ -28,7 +28,7 @@
     "mongoose": "5.x"
   },
   "peerDependencies": {
-    "mongoose": "5.x"
+    "mongoose": "5.x || 6.x"
   },
   "author": "Valeri Karpov <val@karpov.io>",
   "license": "Apache 2.0",

--- a/test/examples.test.js
+++ b/test/examples.test.js
@@ -6,11 +6,16 @@ const updateVersioningPlugin = require('../');
 
 describe('Examples', function() {
   before(function*() {
-    mongoose.set('useFindAndModify', false);
+    let connectOptions;
 
-    yield mongoose.connect('mongodb://localhost:27017/updateversioning', {
-      useNewUrlParser: true
-    });
+    if (/^5\./.test(mongoose.version)) {
+      mongoose.set('useFindAndModify', false);
+      connectOptions = {
+        useNewUrlParser: true
+      };
+    }
+
+    yield mongoose.connect('mongodb://localhost:27017/updateversioning', connectOptions);
 
     yield mongoose.connection.dropDatabase();
   });

